.travis.yml

```diff
@@ -2,21 +2,20 @@ language: node_js
 node_js:
     - "0.12"
     - "0.10"
-#   - "0.8"   # no longer supported by iconv
-#   - "0.6"   # no longer supported by async
-    - "iojs-v1.8.1"
-    - "iojs"
-
+    - "stable"
+    - "iojs-v1"
+    - "iojs-v2"
+    - "iojs-v3"
 services:
     - redis-server
-
-matrix:
-    allow_failures:
-        - node_js: "0.8"
-        - node_js: "iojs"
-#       - node_js: "0.6"
-
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - g++-4.8
+env:
+  - "CXX=g++-4.8"
```

Comment:
Okay, so put a comment there telling why: "these are required for building on node.js 4.0"